src/mixer.c

```diff
@@ -554,29 +571,20 @@ void mixTable(void)
             motor[0] = constrain(rcCommand[THROTTLE], mcfg.minthrottle, mcfg.maxthrottle);
     }
 
-    // airplane / servo mixes
-    switch (mcfg.mixerConfiguration) {
-        case MULTITYPE_CUSTOM_PLANE:
-        case MULTITYPE_FLYING_WING:
-        case MULTITYPE_AIRPLANE:
-        case MULTITYPE_BI:
-        case MULTITYPE_TRI:
-        case MULTITYPE_DUALCOPTER:
-        case MULTITYPE_SINGLECOPTER:
-            servoMixer();
-            break;
-        case MULTITYPE_GIMBAL:
-            servo[0] = (((int32_t)cfg.servoConf[0].rate * angle[PITCH]) / 50) + servoMiddle(0);
-            servo[1] = (((int32_t)cfg.servoConf[1].rate * angle[ROLL]) / 50) + servoMiddle(1);
-            break;
+    if (core.useServo == 1) {
+        // reset all servos
+        resetServos();

```

Comment:
Nope. They're updated into servo[] array here and later committed to hardware via writeServos().
You can however cleanup that gimbal forward thing so its also done in writeServos, isntead of here.